apps/block_scout_web/lib/block_scout_web/templates/bridged_tokens/_tile.html.eex

```diff
@@ -10,6 +10,16 @@
     <div class="centered-container">
       <div>
         <div style="display: table;">
+          <%= if System.get_env("DISPLAY_TOKEN_ICONS") === "true" do %>
+            <% foreign_chain_id = if Map.has_key?(@bridged_token, :foreign_chain_id), do: @bridged_token.foreign_chain_id, else: nil %>
+            <% chain_id_for_token_icon = if foreign_chain_id, do: foreign_chain_id |> Decimal.to_integer() |> to_string(), else: System.get_env("CHAIN_ID") %>
```

Comment:
I guess this is where the github icon js is getting it's input from. In future we could do something similar but serve icons from the app. I don't know enough about bridged tokens or if our implementation of them can be detected by blockscout in the same way as Ethereum